deployment/chart/values.yaml

```diff
@@ -71,25 +71,43 @@ tolerations: []
 
 affinity: {}
 
-ticketsEnvironment: "production"
-ticketsSecretToken: "this-is-a-very-bogus-key"
-ticketsServeStaticFiles: "true"
-ticketsSMTPHost: ""
-ticketsSMTPPort: "" 
-ticketsSMTPDomain: ""
-ticketsSMTPUser: ""
-ticketsSMTPPassword: ""
-ticketsDBSeed: ""
-ticketsHost: ""
-ticketsEmailAddress: ""
-stripeSecretKey: ""
-stripePublicKey: ""
-pumaConcurrency: 2
-pumaThreads: 2
+config:
+  ticketsEnvironment: "production"
+  ticketsSMTPHost: ""
+  ticketsSMTPPort: "" 
+  ticketsSMTPDomain: ""
+  ticketsDBSeed: ""
```

Comment:
That will be false for prod and true for staging

Reply:
_Putting all the reviews for environment configuration into this thread and resolving the others_

There is an `env` directory in here that can apply environment specific values files.
- `base.yaml` is a common configuration for all envs. This could be rolled into the chart's `values.yaml` file.
- `production.yaml` builds upon the base values and is describing the necessary components
- any other env utilizing helm should utilize this pattern

Regarding specific configuration keys/values, I was working through the existing chart and attempting to not break things based off what I saw in the repo. I attempted to remove only the things that I was directly replacing, i.e. postgres. The previous chart required certain values to be defined, the way that I refactored this was so only those configs you explicitly define will be created.

As a fist pass in refactoring this, are accurate settings important? IMO getting the large change in with smaller scoped incremental changes is more effective.